tests/cupy_tests/indexing_tests/test_generate.py

```diff
@@ -6,6 +6,22 @@
 
 
 @testing.gpu
+class TestIndices(unittest.TestCase):
+
+    _multiprocess_can_split_ = True
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_list_equal()
+    def test_indices_list0(self, xp, dtype):
+        return xp.indices((0,), dtype)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_list_equal()
+    def test_indices_list1(self, xp, dtype):
+        return xp.indices((1, 2, 3, 4), dtype)
```

Comment:
This test does not pass, too. Can you please fix it?
It seems you need to separate `dtype=numpy.bool_` case. In that test case, check exception is raised using `@numpy_cupy_raises`.